avocado-setup.py

```diff
@@ -487,27 +489,32 @@ def parse_test_config(test_config_file, avocado_bin, enable_kvm):
                 logger.debug("%s does not exist", test_dic['test'])
                 continue
             if len(line) > 1:
```

Comment:
I guess we need a for loop here to process all items, because current logic misses the line with .yaml and additional args.

Reply:
I see it getting handled in ```if len(line) > 2:
                test_dic['args'] = " --execution-order %s " % line[2]	                arg_flag = 1
                test_dic['args'] = " %s" % line[2]```

but it is better to handle here with additional for loop, what say?

Better to have some comments over the code on what part of config is getting handled for future reference....

Reply:
I am not sure if I understand properly. `for` loop to process all items meaning for each of the additional args or for each line?

Reply:
I was thinking for additional args, it won't appropriate, ignore first comment, adding comment in the code would help for which part for config file is getting processed.